src/components/cardbuilder/cardBuilder.js

```diff
@@ -1082,11 +1082,7 @@ import 'programStyles';
 
                 if (options.showPersonRoleOrType) {
                     if (item.Role) {
-                        lines.push('as ' + item.Role);
-                    } else if (item.Type) {
```

Comment:
@MrTimscampi If you take a look at [this line](https://github.com/jellyfin/jellyfin-web/pull/960/files/60329961b52737ad182a76940065f6cf264c5bcc#diff-265725aca33ce25b93965819e407eb05L1087), you will see that the Types weren't actually used at all in the interface.

``Type`` is what the person does (e.g Composer, Actor, Producer, etc), just as you said.
``Role`` is what the person does in the movie. See this for example:

![image](https://user-images.githubusercontent.com/10274099/78390211-7e4a1900-75e4-11ea-8567-12c7bbfd12f0.png)

In this case, ``Type`` is *Actor* and ``Role`` is *Buster Moon (voice)*

This is completely redundant, because most of the times Role = Type. See this one:

![image](https://user-images.githubusercontent.com/10274099/78390377-bd786a00-75e4-11ea-9fda-be31920dfdd8.png)

``Producer`` is the role and the type. So we would have the same thing twice in the UI. That's why I think this line was removed, because in your example, Beethoven's Type (Composer) = Role (Composer)

If you think that the Type should be back, let's label this as discussion, but I don't think we should: It's easily deductable already.

This is what you mean? If not, can you please attach an screenshot showcasing it? I'm not sure I understand 100% the context.

Reply:
Then we should probably change how the non-actor roles are handled, but no need to do so right now :)